fairroot/online/steer/FairRunOnline.cxx

```diff
@@ -188,13 +187,13 @@ void FairRunOnline::Init()
     }
 
     // --- Get event header from Run
-    if (!fEvtHeader) {
+    if (evtHeader == nullptr) {
         LOG(fatal) << "FairRunOnline::Init() No event header in run!";
         return;
     }
```

Comment:
```suggestion
```

I wonder, whether this is needed at all? `GetEventHEader()` ensures that a header is returned always, right?

Reply:
Yes, you are right.